### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by tcrasset** [Wasilp/DL-wine-quality#1]
# Structured data review

## Good

- REALLY LIKE the README, nice explanation, knew exactly what improved and what not
- Good use of `.gitignore`
- Loss seems to decrease in every model, nice!
- Good accuracy!
- Nice standardization + resampling

## To be improved

- README format and typos should be corrected
- Not sure about the usefulness of having a different activation function for each layer [More info here](https://machinelearningmastery.com/choose-an-activation-function-for-deep-learning/)
- You should spend some time refactoring most of your code into methods (e.g. `resample_df`, `split_dataset`). You have a lot of code duplication across your notebooks. Maybe even a different file/folder for the models, another for the preprocessing, etc...
- Ideally, you should have used a validation set and stopped training when the loss on the validation set stops decreasing
